requirements.txt

```diff
@@ -0,0 +1,12 @@
+astroid==2.4.2
+autopep8==1.5.3
+colorama==0.4.3
```

Comment:
This is the only dependency which was required to run your program from what I could tell